src/main/java/mate.academy/model/Product.java

```diff
@@ -1,12 +1,19 @@
 package mate.academy.model;
 
+import mate.academy.lib.Inject;
+
 import java.math.BigDecimal;
 
 public class Product {
+    @Inject
     private Long id;
+    @Inject
     private String name;
+    @Inject
     private String category;
+    @Inject
     private String description;
+    @Inject
     private BigDecimal price;
```

Comment:
The `@Inject` annotation should not be used on model class fields like `id`, `name`, `category`, `description`, and `price`. It is typically used to mark a dependency that is to be injected into a class by an injector (DI framework), and model classes usually do not have such dependencies.